src/chunk.c

```diff
@@ -1860,3 +1845,52 @@ chunks_return_srf(FunctionCallInfo fcinfo)
 	else						/* do when there is no more left */
 		SRF_RETURN_DONE(funcctx);
 }
+
+static void
+drop_chunks_main(Oid table_relid, Datum older_than_datum, Datum newer_than_datum, Oid older_than_type, Oid newer_than_type, bool cascade)
```

Comment:
shouldn't this be something like `chunk_drop` or `chunk_drop_impl` we usually dont use `main` anywhere

Reply:
I used `main` in the main telemetry function, but happy to change this to something else. How about `do_drop_chunks`? I'm not sure if we have a consistent convention throughout the code (in particular, doesn't look like `_impl` is common either).

Reply:
I used `main` in the main telemetry function, but happy to change this to something else. How about `do_drop_chunks`? I'm not sure if we have a consistent convention throughout the code (in particular, doesn't look like `_impl` is common either).

Reply:
do_drop_chunks works

Reply:
done